### PR TITLE
Feature/thumbnail buttons

### PR DESCRIPTION
**Pull request by cs1m0n**

### CODE REVIEW

File: electron/interface.d.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export interface IElectronAPI {
+  setThumbarBtns: (mode: string) => Promise<void>;
```
Comment (kkuepper) [inline electron/interface.d.ts:2]: can you change the name?
setThumbBarButtons

File: plugins/5.mediaKeys.ts [hunk 2]
```diff
@@ -37,9 +37,11 @@ export default defineNuxtPlugin(() => {
         switch (state) {
           case MediaPlayerStatus.Paused:
             navigator.mediaSession.playbackState = "paused";
+            window.electronAPI.setThumbarBtns("paused");
```
Comment (kkuepper) [inline plugins/5.mediaKeys.ts:40]: in the browser electronAPI shouldn't exist. So we need to check if that object exists.
Maybe window.electronAPI?. is enough?

Comment (cs1m0n) [reply plugins/5.mediaKeys.ts]: Yes, tested it. That's enough.

File: electron/main.ts [hunk 3]
```diff
@@ -40,6 +49,54 @@ const openWindow = (url: string) => {
     titleBarStyle: process.platform === "darwin" ? "hidden" : "default",
   });
 
+  const previousTrack = {
+    tooltip: "Previous",
```
Comment (kkuepper) [inline electron/main.ts:53]: Maybe we should translate this but it's probably difficult in this file.
Can you add a //ToDo: figure out how to translate the tooltip

**Review by kkuepper — APPROVED** [review]

**Pull request merged**